### PR TITLE
Add new GitHub repository templates

### LINKED ISSUE #4: Add Template for Pull Request
The template must have the following parts:
- Title: Pull Request Overview
- Summary
- Changes Made
- Tests Results
- Next Steps

Use the following photo as an example:
Screenshot 2024-10-29 at 11 36 24 AM

### PR DESCRIPTION
**Pull request by nov0ax**
## Summary by CodeRabbit

- **New Features**
	- Introduced a standardized pull request template to enhance clarity and consistency in submissions.
	- The template includes sections for overview, changes made, test results, and next steps, facilitating better communication among contributors.

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+# Pull Request Overview
+
+## 📝 Summary
+ Provide a general overview of your changes that this PR accomplishes.
+
```
Comment (jimenezz22) [inline .github/PULL_REQUEST_TEMPLATE.md:5]: please add:
### Related Issues
- Closes #(issue number) Replace with actual issue number.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+## 🛠️ Changes Made
+Mark with an `x` all the checkboxes that apply (like `[x]`).
+
+- [ ] 📝 Documentation (updates to README, docs, or comments)
+- [ ] 🐛 Bug fix (non-breaking change which fixes an issue)
+- [ ] 👌 Enhancement (non-breaking change which adds functionality)
+
+### 💡Changes summary
```
Comment (jimenezz22) [inline .github/PULL_REQUEST_TEMPLATE.md]: change to: Changes Made

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
+# Pull Request Overview
+
+## 📝 Summary
+ Provide a general overview of your changes that this PR accomplishes.
+
+## 🛠️ Changes Made
```
Comment (jimenezz22) [inline .github/PULL_REQUEST_TEMPLATE.md]: change to: Type of Change

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+Mark with an `x` all the checkboxes that apply (like `[x]`).
+
+- [ ] 📝 Documentation (updates to README, docs, or comments)
+- [ ] 🐛 Bug fix (non-breaking change which fixes an issue)
+- [ ] 👌 Enhancement (non-breaking change which adds functionality)
+
+### 💡Changes summary
+Provide a clear and concise description of what you changed and why.
```
Comment (jimenezz22) [inline .github/PULL_REQUEST_TEMPLATE.md:17]: Replace and add this points:

### What's Changed
Provide a clear and concise description of what you changed and why.

### Implementation Details
Describe how you implemented the changes.

### Technical Notes
Include any technical details that reviewers should be aware of.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+
+### ✅ Tests Results
+Describe the tests you performed to verify your changes:
+
+### Test Coverage
+<!-- Mark with an `x` all the checkboxes that apply (like `[x]`). -->
+- [ ] ✅ Unit Tests
+- [ ] ✅ Integration Tests
+- [ ] ✅ Manual Testing
+
```
Comment (jimenezz22) [inline .github/PULL_REQUEST_TEMPLATE.md:32]: Add:

### Evidence
Provide relevant evidence of testing (screenshots, test outputs, etc.).

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+### 🔍 Testing Notes
+Include any special testing considerations or edge cases checked.
+
+## ⏭️ Next Steps
+Indicate actions or improvements to be taken after this PR, if applicable.
+- [ ] Action item 1
```
Comment (jimenezz22) [inline .github/PULL_REQUEST_TEMPLATE.md]: remove

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 7]
```diff
@@ -0,0 +1,38 @@
...
+- [ ] Action item 2
```
Comment (jimenezz22) [inline .github/PULL_REQUEST_TEMPLATE.md]: remove

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 8]
```diff
@@ -0,0 +1,38 @@
...
+
+---
+**Notes:**
```
Comment (jimenezz22) [inline .github/PULL_REQUEST_TEMPLATE.md]: remove

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 9]
```diff
@@ -0,0 +1,38 @@
...
+- 👥 Tag relevant team members for review
```
Comment (jimenezz22) [inline .github/PULL_REQUEST_TEMPLATE.md]: remove

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 10]
```diff
@@ -0,0 +1,38 @@
...
+
+ ### 🔗 Related Issues
```
Comment (jimenezz22) [inline .github/PULL_REQUEST_TEMPLATE.md]: remove

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 11]
```diff
@@ -0,0 +1,38 @@
...
+- Closes #(issue number) *Replace with actual issue number*.
```
Comment (jimenezz22) [inline .github/PULL_REQUEST_TEMPLATE.md]: remove

**Review by jimenezz22 — CHANGES_REQUESTED** [review]

**Review by jimenezz22 — APPROVED** [review]
